FirebaseAdmin/FirebaseAdmin/Auth/Providers/OidcProviderConfig.cs

```diff
@@ -71,33 +72,16 @@ internal OidcProviderConfig(Request request)
         public string ClientSecret { get; }
 
         /// <summary>
-        /// Gets what response types are being used for this OIDC config.
+        /// Gets a value indicating whether an ID Token response type will be provided.
         /// </summary>
-        public ResponseTypeArgs ResponseType { get; }
+        public bool IDTokenResponseType { get; }
```

Comment:
Sorry for not spotting this earlier. But I think this ought to be `IdTokenResponseType`? We currently have APIs with names like `VerifyIdTokenAsync()`. So this should follow that naming convention.

Same at `ResponseTypeInfo.IdToken`.